tutorials/peft-curation/main.py

```diff
@@ -163,7 +165,15 @@ def run_curation_pipeline(args: Any, jsonl_fp: str) -> str:
 def main():
     parser = argparse.ArgumentParser()
     parser = add_distributed_args(parser)
+    parser.add_argument(
+        "--input-meta",
```

Comment:
Since we haven't run into any issues in the tutorial datasets, and for the clarity of the tutorials, I think you should omit the argument from them.